From 2f2791f8959c9d4407936cda166a2cba796dce12 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 11 Nov 2022 07:14:59 +0800
Subject: [PATCH] Revert "Linux 5.10.154"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit f5b40c0eb9ea3d8233b9a2e9af6784c81204d454.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 43fecb404581..d1cd7539105d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 154
+SUBLEVEL = 153
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

